### DISCUSSION TITLE
Can I remove delete prompt?

### DISCUSSION THREAD

**Discussion by shirobachi** [TriliumNext/Trilium#3568] [Q&A] [answered]
I would like to ask if I am able to remove delete prompt question and how, if so?
Especially for empty files or/and without children pages

![image](https://user-images.githubusercontent.com/16509749/215340745-cf2ad96b-01b9-461c-8f67-66c0ae92e114.png)

Thanks in advance!
Simon

**Comment by zadam** [comment, marked as answer]
It's not possible. But confirming the dialog is just an enter away, that shouldn't be that big of a problem?

**Reply by shirobachi** [reply]
Sorry maybe you're right, I am just prefer shortcuts
